src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -130,7 +131,19 @@ def get_scaleset_principal_id() -> UUID:
         credential=get_identity(), subscription_id=get_subscription()
     )
     uid = client.resources.get_by_id(get_scaleset_identity_resource_path(), api_version)
-    return UUID(uid.properties["principalId"])
+
+    # workaround issue from azure-mgmt-resource, where properties is now a str
+    # instead of an obj.
+    # https://github.com/Azure/azure-sdk-for-python/pull/18686/files
```

Comment:
Was this meant to link to something more specific, or is the `/files` stray?

Reply:
It was more of a "look at the massive # of `object` that are now `str` in this PR"